### DISCUSSION TITLE
Adding days to a date in JS using moment

### DISCUSSION THREAD

**Discussion by SICUFI** [Budibase/budibase#10473] [Help] [answered]
Hello,

I'm trying to add days to a date using moment.js .add function, but I can't get it to work.
I guess it's some sort of syntax issue, but I can't find a suitable example to learn from
Can someone please help? This is the code I'm using :

```
var timeexprw1 = helpers.date($("now"), "HH.mm.ss", "Europe/Rome");
var finaldate = helpers.date(timeexprw1).add(2, 'h');
return finaldate;
```

Thank you very much

**Comment by ConorWebb96** [comment, marked as answer]
Hey @SICUFI,

The date.helper doesn't have the function within to do what you are wanting, its limited to time zoning and formatting the date. You can however achieve this with something similar to the following:
<img width="638" alt="Screenshot 2023-05-04 at 08 58 27" src="https://user-images.githubusercontent.com/126772285/236144667-a616650d-808c-4afb-9a88-2d03098c19bd.png">

**The result can be seen below:**
<img width="560" alt="Screenshot 2023-05-04 at 09 00 57" src="https://user-images.githubusercontent.com/126772285/236145000-9a1577bb-85e4-4806-9ed8-05616793d3c4.png">

I hope this helps!

**Reply by SICUFI** [reply]
Thank you,

managed to get it to work :)
